src/psyclone/tests/psyGen_test.py

```diff
@@ -32,7 +32,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Authors: R. W. Ford, A. R. Porter, S. Siso and N. Nobre, STFC Daresbury Lab
-# Modified I. Kavcic and L. Turner, Met Office
+# Modified I. Kavcic, L. Turner and O. Brunt, Met Office
```

Comment:
I would suggest adding `:` after `# Modified` for consistency with the "Authors" line above.